<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulatorConfig</class>
 <widget class="QDialog" name="EmulatorConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emulator Configuration</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="paramsEdit">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>108</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>113</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QComboBox" name="emulatorBox">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>63</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>68</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Emulator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emuMgrButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to the Emulator Manager</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
